lib/sprinkle/installers/source.rb

```diff
@@ -47,12 +47,16 @@ module Installers
     #     source 'http://magicbeansland.com/latest-1.1.1.tar.gz' do
     #       prefix    '/usr/local'
     #
-    #       pre :prepare { 'echo "Here we go folks."' }
-    #       post :extract { 'echo "I believe..."' }
-    #       pre :build { 'echo "Cross your fingers!"' }
+    #       pre :prepare, 'echo "Here we go folks."'
+    #       post :extract, 'echo "I believe..."'
+    #       pre :build, 'echo "Cross your fingers!"'
```

Comment:
Please leave the {} deferred examples unless you want to document the difference and when someone should use one over the other.